Travis build fails suddenly due to not being able to verify Cloud SDK version

appengine-plugins-core doesn't seem to be able to verify the version of the installed Cloud SDK all of a sudden. That's puzzling. BTW, I've cleared the Travis cache, but for a different reason.

```
testStage(com.google.cloud.tools.eclipse.appengine.deploy.standard.StandardStagingDelegateTest)  Time elapsed: 5.726 sec  <<< FAILURE!
java.lang.AssertionError: Could not create project
	at com.google.cloud.tools.appengine.cloudsdk.CloudSdk.validateCloudSdkVersion(CloudSdk.java:490)
	at com.google.cloud.tools.appengine.cloudsdk.CloudSdk.validateCloudSdk(CloudSdk.java:477)
	at com.google.cloud.tools.eclipse.appengine.facets.AppEngineFacetInstallDelegate.validateAppEngineJavaComponents(AppEngineFacetInstallDelegate.java:43)
	at com.google.cloud.tools.eclipse.appengine.facets.AppEngineFacetInstallDelegate.execute(AppEngineFacetInstallDelegate.java:36)
	at com.google.cloud.tools.eclipse.appengine.facets.StandardFacetInstallDelegate.execute(StandardFacetInstallDelegate.java:50)
	at org.eclipse.wst.common.project.facet.core.internal.FacetedProject.callDelegate(FacetedProject.java:1477)
	at org.eclipse.wst.common.project.facet.core.internal.FacetedProject.modifyInternal(FacetedProject.java:441)
	at org.eclipse.wst.common.project.facet.core.internal.FacetedProject.mergeChangesInternal(FacetedProject.java:1181)
	at org.eclipse.wst.common.project.facet.core.internal.FacetedProject.access$2(FacetedProject.java:1117)
	at org.eclipse.wst.common.project.facet.core.internal.FacetedProject$1.run(FacetedProject.java:324)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2241)
	at org.eclipse.wst.common.project.facet.core.internal.Facet[INFO] 
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -7,7 +7,7 @@ install:
   # use mvn ≥ 3.3.9 to ensure faulty test exit states fail the build (#1276)
   - mvn -B -N io.takari:maven:wrapper -Dmaven=3.5.0
   # download and install Cloud SDK (may be cached)
-  - build/install-cloudsdk.sh 171.0.0 $HOME
+  - build/install-cloudsdk.sh 183.0.0 $HOME
 # Tycho ≥ 0.26 requires Java 8
 jdk: oraclejdk8
 env:
@@ -19,7 +19,7 @@ env:
     - JAVA_RUNTIME=JavaSE-1.8 ECLIPSE_TARGET=neon
     - JAVA_RUNTIME=JavaSE-1.8 ECLIPSE_TARGET=oxygen MAVEN_FLAGS=-Pjacoco
   global:
-    - PATH=$PWD/google-cloud-sdk/bin:$PATH
+    - PATH=$HOME/google-cloud-sdk/bin:$PATH
     # Our maven build typically reports about 512m
     - MAVEN_OPTS=-Xmx700m
     - CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true

